### DISCUSSION TITLE
backing up or exporting current stock Sv06 plus firmware

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#126] [Q&A]
Is there a way to back up before flashing clipper?

**Comment by bassamanator** [comment]
Never looked into but I'm sure it's possible to pull the bin from the board. Backups of bin are really unnecessary considering you can just download the bin from sovol.

**Reply by dasb00ter** [reply]
Thanks for taking the time to answer. It just appears from my research that Sovol is a little disorganised with their firmwares. I've seen threads on the sovol site discussing machines sent out with newer firm wares than available on the website, an odd decision in terms of numbering systems of versions etc etc. Was hoping for a command line solution without soldering and pulling firmware off bins

**Comment by bassamanator** [comment]
`avrdude` will *likely* allow you to accomplish this.

In any case, just find out from Marlin users what's the latest firmware and download it, and you won't need to figure out how to pull firmware from the motherboard.
